src/components/ProductForm/ProductForm.tsx

```diff
@@ -0,0 +1,156 @@
+import { LoaderOverlay } from 'components/LoaderOverlay/LoaderOverlay';
+import { ReactElement } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPen } from '@fortawesome/free-solid-svg-icons';
+import { Dropdown } from 'components/Dropdown/Dropdown';
+import { useProductFormContext } from './ProductFormContext';
+import { Button } from 'components/Buttons/Button';
+
+export const ProductForm = (): ReactElement => {
+  const formContext = useProductFormContext();
+  const [formProduct, setFormProduct] = formContext.useFormProduct;
+
+  //Placeholder Data
+  interface ITag {
+    id: number;
+    name: string;
+  }
+  const allTags: ITag[] = [
+    { id: 0, name: 'Vegan' },
+    { id: 1, name: 'Vegetarian' },
+    { id: 2, name: 'Good Food' },
+    { id: 3, name: 'Nut-Free' },
+    { id: 4, name: 'Gluten-Free' },
+    { id: 5, name: 'Breakfast Menu' },
+    { id: 6, name: 'Vegan' },
+    { id: 7, name: 'Vegetarian' },
+    { id: 8, name: 'Good Food' },
+    { id: 9, name: 'Nut-Free' },
+    { id: 10, name: 'Gluten-Free' },
+    { id: 11, name: 'Breakfast Menu' },
+    { id: 12, name: 'Vegan' },
+    { id: 13, name: 'Vegetarian' },
+    { id: 14, name: 'Good Food' },
+    { id: 15, name: 'Nut-Free' },
+    { id: 16, name: 'Gluten-Free' },
+    { id: 17, name: 'Breakfast Menu' },
+    { id: 18, name: 'Vegan' },
+    { id: 19, name: 'Vegetarian' },
+    { id: 20, name: 'Good Food' },
+  ];
+  const categories = ['Mains', 'Sides', 'Drinks'];
+
+  const toggleTag = (event: React.ChangeEvent<HTMLInputElement>, toggleTagId: number): void => {
+    if (event.target.checked) {
+      setFormProduct({
+        ...formProduct,
+        tags: formProduct.tags.concat([toggleTagId]),
+      });
+    } else {
+      setFormProduct({
+        ...formProduct,
+        tags: formProduct.tags.filter((tagId) => tagId !== toggleTagId),
+      });
+    }
+  };
+
+  return (
+    <>
+      <form className="w-full overflow-auto">
+        <div className="grid grid-cols-4 gap-4 p-2">
+          <div className="col-span-2 row-span-3 flex h-full w-full items-center justify-center bg-gray-200">
+            <FontAwesomeIcon icon={faPen} />
+          </div>
+          <div className="col-span-2">
+            <label className="italic text-gray-400" htmlFor="product_create_form_name">
```

Comment:
is text-gray-400 correct?

Reply:
I've just been using built in colours, whatever is closest to the figma, they're probably all a bit off. Can you scrub RGB values from the figma? Also the colours used across the site should probably be defined in one place.

Reply:
Yes you're right about colours needing to be defined it one place.

And yes you can, by clicking the text and hitting inspect on the right panel.

Reply:
we could look into extending `tailwind.js` to add our own colour classes